docs/user-guides/team/rbac.md

```diff
@@ -45,6 +58,23 @@ spec:
         - "list"
 ```
 
+## Seeded default TeamRoles
+
+Greenhouse provides a set of [default `TeamRoles`](./../../../pkg/controllers/organization/teamrole_seeder_controller.go) that are seeded to all clusters:
+
+- `cluster-admin`
+  - full priviledges
+- `cluster-viewer`
+  - `get`, `list` and `watch` all resources
+- `cluster-developer`
+  - Aggregated role. Greenhouse aggregates the `application-developer` and the `cluster-viewer`. Further `TeamRoles` can be aggregated.
+- `application-developer`
+  - Set of permissions on `pods`, `deployments` and `statefulsets` necessary to develop applications on k8s
+- `node-maintainer`
+  - `get` and `patch` `nodes`
+- `namespace-admin`
+  - all permissions on `namespaces`
```

Comment:
As discussed in the dev call it would be nice if these were visualised as markdown tables.

Reply:
done